omod/src/main/resources/config.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<module configVersion="1.4">
+
+	<id>teleconsultation</id>
+	<name>Teleconsultation</name>
```

Comment:
@gsluthra @angshu Some how I'm unable to add these as variables and interpolate them.
Refer [commit](https://github.com/Bahmni/openmrs-module-teleconsultation/pull/1/commits/fe7c490f9d5d342cf09c339cb502f63522a466e3)